packages/client/.eslintrc

```diff
@@ -15,6 +15,12 @@
     "react/state-in-constructor": "off",
     "react/jsx-props-no-spreading": "off",
     "react/function-component-definition": "off"
+    // "import/no-extraneous-dependencies" : [
```

Comment:
Just remove this commented code🙏🏽

Reply:
Done

Reply:
I can still see the commented code, maybe you forgot to push it? 🤔

Reply:
Yes!